.github/CODEOWNERS

```diff
@@ -77,6 +77,9 @@ Jenkinsfile.github                                       @Cray-HPE/csm-devops
 # IUF Items
 /operations/iuf/                                         @Cray-HPE/IUF
 
+#Observability Items
+/operations/observability                                @Cray-HPE/observability
```

Comment:
This team does not exist.
I think this should be `@Cray-HPE/cray-monitoring`

https://github.com/orgs/Cray-HPE/teams/cray-monitoring

Reply:
@rustydb sure, I have made this change